checksig/checksig.sh

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env bash
+# SPDX-FileCopyrightText: 2024 Technology Innovation Institute (TII)
+# SPDX-License-Identifier: Apache-2.0
+
+# Stop on error
+set -e
+
+# Required installed commands
+PREREQUISITES=( "python3 --version" "wget -V" "tar --version" "xxd -v" "openssl version" "find --version" "mktemp --version" "realpath --version" "chmod --version" "grep --version" "mv --version" "rm --version" )
+
+# URL prefix for the build reports
+REPURL="https://vedenemo.dev/files/build_reports"
+
+# URL for the sha256tree.py
+SHA256URL="https://raw.githubusercontent.com/tiiuae/ci-public/main/sha256tree/sha256tree.py"
+
+# URL for the server keys
+KEYURL="https://vedenemo.dev/files/keys/"
```

Comment:
not related to this review, but for future versions, would it be a good idea to move these settings to a separate file somewhere in KEYDIR, to protect them from being modified in case of localhost compromise?

Reply:
Seems to me, that it would just complicate things more than increase security.
It could be argued, that it's better that these settings are downloaded with the script, as then localhost compromiser needs to do more than just change the files in .ghafkeys.
Keep in mind that this is supposed to be something end user can just quickly download and easily check signatures for some packages.